src/components/InterruptedVideosGallery.tsx

```diff
@@ -15,12 +15,25 @@ const InterruptedVideosGallery: React.FC<RouteComponentProps> = () => {
 
```

Comment:
Is ContinueWatchingVideos a better name for this? InterruptedVideos sounds a bit odd doesn't it?

Reply:
That's the name we're using internally in other places as well, the `__typename` will have `INTERRUPTED` value. You may be right, but I don't want to do that as part of this PR, it's out of scope tbh

Reply:
yeah, it's a bit unrelated to the PR

Everything else looks good to me, tho not sure how can this be tested

Reply:
I guess netlify build logs can be useful: https://app.netlify.com/sites/atlas-development/deploys/603ce31945fe680007054178

Also you can checkout the branch locally and change the `REACT_APP_ENV` in `.env` to see how the app behaves

Reply:
for each deploy we will get this in the netlify log:
```
1:52:59 PM: Using STAGING variables
1:52:59 PM: Exposing STAGING_ORION_URL as REACT_APP_ORION_URL=https://orion-staging.joystream.app/graphql
1:52:59 PM: Exposing STAGING_QUERY_NODE_URL as REACT_APP_QUERY_NODE_URL=https://hydra-staging.joystream.app/server/graphql
1:52:59 PM: Exposing STAGING_STORAGE_NODE_URL as REACT_APP_STORAGE_NODE_URL=https://staging-3.joystream.app/storage/asset/v0/
1:52:59 PM: Exposing REACT_APP_ENV=STAGING
```